spec/models/mixins/process_tasks_mixin_spec.rb

```diff
@@ -257,12 +257,12 @@ def test_method
           expect(api_collection).to receive(:find).with(0).and_return(Struct.new(:id).new(0))
           expect(api_collection).to receive(:find).with(1).and_return(double("Something that responds", :id => 0, :the_task => nil))
 
-          expect($log).to receive(:error).with(a_string_including("undefined method `the_task' for #<struct id=0>")).and_call_original
+          expect($log).to receive(:error).with(a_string_matching(/undefined method `the_task' for/)).and_call_original
```

Comment:
This doesn't really use a regex so what about:
```suggestion
          expect($log).to receive(:error).with(a_string_including("undefined method `the_task' for")).and_call_original
```

Reply:
Same with below